lib/galaxy/files/templates/examples/production_onedata.yml

```diff
@@ -0,0 +1,41 @@
+- id: onedata
+  version: 0
+  name: Onedata Storage
+  description: |
+    This template allows connecting to Onedata servers.
+
+    Onedata is a global data management system providing a unified data access
+    interface across different storage providers. More information on Onedata
+    can be found on [Onedata's website](https://onedata.org/#/home).
+  variables:
+    onezone_domain:
+      label: Onezone domain
+      type: string
+      help: Domain of the Onezone service (e.g. datahub.egi.eu) to connect to.
+    disable_tls_certificate_validation:
+      label: Disable tls certificate validation?
+      type: boolean
+      help: |
+        Allows connection to Onedata servers that do not present trusted SSL certificates.
+        SHOULD NOT be used unless you really know what you are doing.
+      default: False
+    writable:
+      label: Writable?
+      type: boolean
+      help: |
+        Allow Galaxy to write data to this Onedata storage.
+        Requires access token with write data access.
+      default: False
+  secrets:
+    access_token:
+      label: Access Token
+      help: |
+        Your [access token](https://onedata.org/#/home/documentation/topic/stable/tokens),
+        suitable for REST API access in a Oneprovider service. May be confined
+        to read-only data access.
```

Comment:
May be confined to read-only data access, unless you wish to use the "remote export" tool (write permissions are needed then).

(jeśli ten tool się nie nazywa "remote export" to trezba zaktualizować)